cmd/algocfg/profileCommand.go

```diff
@@ -69,7 +69,9 @@ var (
 	relay = configUpdater{
 		description: "Relay consensus messages across the network and support catchup.",
 		updateFunc: func(cfg config.Local) config.Local {
-			cfg.Archival = true
+			cfg.MaxBlockHistoryLookback = 20000
```

Comment:
So with this, non-archivals are 1000 blocks, relays are 20000, and archivals are all? Just clarifying we have three typical options for retention after this.

Reply:
The min lookback due to system requirements is 1000+321 if I am not mistaken. Setting MaxBlockHistoryLookback (defaults to 0) is what gives you different values for relays. For nodes set to be archival, this setting has no effect.